tests/integration/suite/daprd/conversation/http/basic.go

```diff
@@ -43,15 +43,14 @@ func (b *basic) Setup(t *testing.T) []framework.Option {
 apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
-  name: conversation
+  name: echo
```

Comment:
nit: remove alias import of `procdaprd`.

Reply:
Done